example/fixtures.py

```diff
@@ -38,6 +40,15 @@ def exec_find(self, params):
     def for_fixture(
             cls, request, name, wait_lines, command=None, env_extra={},
             publish_port=True, wait_timeout=None):
+        pods = request.getfixturevalue('pods')
+        # FIXME: there are probably better ways to skip these tests
+        if pods:
+            if request.fixturename in [
+                    'web_only_container', 'single_container']:
+                pytest.skip()
```

Comment:
Maybe `pytest.skip("these tests don't make sense for pods")` and something equivalent in the other branch?